src/service/OWNERS

```diff
@@ -0,0 +1,11 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      pattern: 'video-*.js',
```

Comment:
Should we move these files into the `src/service/video/` folder? The fewer filename patterns the better.

IMO this is a good chance to start massaging our source dir structure into something more sane. Also it would enable semantically grouped subfolder README.md files.

Reply:
Some aren't strictly for the `service` but rather utilities for component implementations.

I agree with the sentiment, though. How about just `src/video`? Maybe we can move the service implementation into that dir as well.

I'm going to go ahead merging this to unblock another PR but I'll restructure later. Thanks for the suggestions.

Reply:
We do have a convention that service classes (objects shared across multiple binaries) are in `src/service/` e.g. our linter bans direct imports. 

We can change this -- just pointing it out.